src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -66,6 +66,32 @@
  * <pre>
  * &lt;module name="NeedBraces"/&gt;
  * </pre>
+ * <p>Valid code example:</p>
+ * <pre>
+ * if (true) { // OK
+ *    return true;
+ * }  else { //OK
+ *    return false;
+ * }
+ * for(int i =0; ; ) { // OK
```

Comment:
please format code correctly and make it more look like real life code (`(int i = 0; i < 5; i++)`

Reply:
done.

Reply:
Not done, code is not formatted

Reply:
I formatted it with 4 spaces.